_pages/en_US/homebrew-launcher-(super-skaterhax).txt

```diff
@@ -64,6 +63,9 @@ If you haven't already, make sure you have a working Internet connection set up
 1. Press (Home) to return to the HOME Menu, then immediately press (A) to launch the browser again
 1. Select the "GO GO!" button on the top screen
   + If any prompts appear, approve all of them
+  + If your device freezes on a yellow screen, hold the POWER button until it turns off, then retry this section
+  + If your device freezes on a red screen, hold the POWER button until it turns off, redo step Step 3 of Section II, then retry this section
+    + If you get another error, [follow this troubleshooting guide](troubleshooting#homebrew-launcher-super-skaterhax)
```

Comment:
If I recall correctly, level 2 nested subbullets do not work in Jekyll. Make it line up with the other bullets

Reply:
accident